overview.js

```diff
@@ -478,30 +478,33 @@ var dtpOverview = Utils.defineClass({
          });
          Main.overview._overview.add_action(this._clickAction);
 
-        Main.overview.viewSelector.appDisplay._views.forEach((v) => {
-            if(v.view._swipeTracker) {
+        [
+            Main.overview.viewSelector._workspacesDisplay,
+            Main.overview.viewSelector.appDisplay._views[1].view
+        ].forEach(v => {
```

Comment:
We need `Main.overview.viewSelector.appDisplay._views[0].view` right?

Reply:
Actually no: I made a mistake adding it in 7e39ff2.

As in GNOME Shell 3.36.5, `Main.overview.viewSelector.appDisplay._views[0].view` has no `_swipeTracker` nor `_panAction`, thus [this change](https://github.com/home-sweet-gnome/dash-to-panel/commit/7e39ff220e8b07e2b67ee4d5d50582a0e68f7966#diff-f4b770fb63521a3f94b12ac51cbf5340L480-R508) was useless.

I'm sorry for the confusion I caused.